src/components/ContributionForm/ContributionForm.tsx

```diff
@@ -83,30 +117,19 @@ const ContributionForm = ({ submit }: { submit: (data: any) => void }) => {
       </RadioGroup>
       {wish === 'Ajout de données' && (
         <>
-          <RadioGroup
-            legend="Vous voulez ajouter"
-            name="additionWish"
-            isInline
-            required
-            value={additionWish}
-            onChange={setAdditionWish}
-          >
-            <Radio label="tracé du réseau" value="Tracé du réseau" />
-            <Radio
-              label="périmètre de la zone de développement prioritaire"
-              value="Périmètre de la zone de développement prioritaire"
-            />
-            <Radio
-              label="tracé d'une extension prévue du réseau"
-              value="Tracé d'une extension prévue du réseau"
-            />
-            <Radio
-              label="informations tarifaires"
-              value="Informations tarifaires"
-            />
-            <Radio label="autre" value="autre" />
-          </RadioGroup>
-          {additionWish === 'autre' && (
+          <CheckboxGroup legend="Vous voulez ajouter" isInline required>
+            {additionWishValues.map((value) => (
+              <Checkbox
+                key={value}
+                label={value}
+                id={value}
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore: Create proper type
```

Comment:
Why ?

Reply:
Because, when I add an 'onClick' prop on the `<Checkbox/>` (from react-DSFR), I got a type error.